www/_data/fetched-files.yml

```diff
@@ -1,153 +1,146 @@
+# REFERENCE ENTRY:
+#
+# -
+#     src:                                                            # required
+#         repoName:    "apache/cordova-cli"                           # required
+#         packageName: "cordova"                                      # if empty: second part of repoName
+#         commit:      "master"                                       # if empty: latest in NPM
+#         path:        "relative/path/to/file/in/repo.md"             # if empty: "README.md"
+#     dest:                                                           # required
+#         permalink:   "/absolute/path/on/cordova/site/index.html"    # if empty: "/gen/" + packageName + "/"
+#
+
 -
     src:
         repoName: "apache/cordova-cli"
         packageName: "cordova"
+        commit: "master"
         path: "doc/bash.md"
-    dest: "cordova-cli/index.md"
+    dest:
+        permalink: "/docs/en/dev/cordova-cli/index.html"
 -
     src:
-        repoName: apache/cordova-plugin-battery-status
-        packageName: cordova-plugin-battery-status
-        displayName: Battery Status
-        description: Monitor the status of the device's battery.
-    dest: cordova-plugin-battery-status/index.md
+        repoName: "apache/cordova-plugin-battery-status"
+        packageName: "cordova-plugin-battery-status"
+    dest:
+        permalink: "/docs/en/dev/cordova-plugin-battery-status/index.html"
```

Comment:
Each string here is custom and totally up to change. For example the `/docs/en/dev/` part will have to go once we have translation. I figured the `plugin` part would be redundant since the word is already in the plugin's ID. I can change it to `plugin` though, if you'd prefer. I don't have much of a preference.